tools/draining/hacheck.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+)
+
+const DEFAULT_HACHECK_PORT = 6666
+const DEFAULT_HACHECK_HOST = "localhost"
```

Comment:
If this is going to be run in pod 99% of the time, this needs to be the yocalhost ip, right?